prov/verbs/src/ep_rdm/verbs_av_ep_rdm.c

```diff
@@ -157,6 +161,23 @@ static int fi_ibv_rdm_av_insert(struct fid_av *av_fid, const void *addr,
 			FI_INFO(&fi_ibv_prov, FI_LOG_AV,
 				"fi_av_insert: bad addr #%i\n", i);
 
+			if(av->eq && (av->flags & FI_EVENT))
+			{
```

Comment:
coding nit: please add space after if-statement - if (blah blah) - and place opening brace at end of line.

You can drop the av->eq portion of this check, as it's covered by the check added above.